projects/knora-ui/src/lib/viewer/values/color-value/color-picker/color-picker.component.spec.ts

```diff
@@ -98,7 +99,7 @@ describe('ColorPickerComponent', () => {
     expect(testHostComponent.form.controls.colorValue.value).toEqual('');
   });
 
-  fit('should be disabled when the readonly input is set to true', () => {
```

Comment:
Oh yes, that was the script searching for `fit` :-)

Reply:
but it has been fixed in the last commit...

Reply:
I know :-) I was just happy that the script actually worked ;-)